google/cloud/spanner/keys.h

```diff
@@ -202,8 +246,65 @@ class KeySet {
   bool IsAll() const { return all_; }
 
  private:
+  template <typename RowType>
+  friend class KeySetBuilder;
+
+  friend ::google::spanner::v1::KeySet internal::ToProto(KeySet const& keyset);
+
   explicit KeySet(bool all) : all_(all) {}
 
+  class ValueRow {
+   public:
+    template <std::size_t SIZE>
```

Comment:
I think non-type template params should be spell like `Size`, just like template params. But in this case, I think I'd just avoid the situation and use `N`.

Reply:
Done.